src/services/print.js

```diff
@@ -271,7 +271,7 @@ ngeo.Print.prototype.encodeWmsLayer_ = function(arr, opacity, url, params) {
   goog.object.remove(customParams, 'FORMAT');
 
   var object = /** @type {MapFishPrintWmsLayer} */ ({
-    baseURL: url,
+    baseURL: url.match(/^https?:/) ? url : 'http:' + url,
```

Comment:
This assume that, if `url` doesn't start with `http` or `https` it will start with `//`. That doesn't sound correct to me. What if `url` looks like `path/to/wms` or `/path/to/wms`?

GeoExt's print provider [includes](https://github.com/geoext/geoext/blob/master/lib/GeoExt/data/PrintProviderBase.js#L633-L647) a `getAbsoluteUrl` function that we could copy to ngeo. The function would be simpler in ngeo as we don't need to support IE < 9.